### DISCUSSION TITLE
Plugnmeet-livekit-1 container unhealthy issue

### DISCUSSION THREAD

**Discussion by ThilinaV98** [mynaparrot/plugNmeet-server#427] [Q&A] [answered]
I tried to install plugnemeet on my server and the installation stops and printing dots. The error is showing as 
**"Container plugnmeet-livekit-1    Error 
dependency failed to start: container plugnmeet-livekit-1 is unhealthy"**

How to fix this issue ??

**Comment by jibon57** [comment]
Try to check the log for livekit. Or you can start composer manually by:
```
cd /opt/plugNmeet
docker compose down
docker compose up
```
This way you'll be able to detect the problem.

**Reply by ThilinaV98** [reply]
![Screenshot from 2024-02-21 10-25-30](https://github.com/mynaparrot/plugNmeet-server/assets/86871304/a044404e-7216-4855-a982-8cec235f570b)

I tried the solution you provided but it did not solve the problem and a screenshot is attached above

**Reply by jibon57** [reply, marked as answer]
In latest version of livekit had removed those 2 fields. So, you can open `livekit.yaml` file & remove:
```
signal_relay:
  enabled: true
psrpc:
  enabled: true
```

**Reply by ThilinaV98** [reply]
Thank you it works.
